BaseLib/FileFinder.h

```diff
@@ -36,7 +36,8 @@ class FileFinder
     FileFinder()
     {
         addDirectory(".");
-        addDirectory(BuildInfo::source_path + "/FileIO");
+        addDirectory(BuildInfo::source_path + "/GeoLib/IO/XmlIO");
```

Comment:
That is *not* specific usage. FileFinder is designed so config files can be found even though they might be at different locations at a different time. XSD files are such config files, so they are added here. If you need to search different directories, feel free to add them.